flagparse/flags.go

```diff
@@ -32,49 +32,36 @@ import (
 const (
 	ConfigFile = "configFile"
 
-	Files        = "files"
-	DefaultFiles = ""
+	Files = "files"
 )
 
 //UploadFileConfig describes file config of uploadable
 type UploadFileConfig struct {
```

Comment:
This struct is not "file" config only. It's a generic upload configuration description. What about: UploadConfig?